packages/trackerless-network/generated/packages/dht/protos/PeerDescriptor.ts

```diff
@@ -0,0 +1,111 @@
+// @generated by protobuf-ts 2.9.4 with parameter server_generic,generate_dependencies,long_type_number
+// @generated from protobuf file "packages/dht/protos/PeerDescriptor.proto" (package "peerDescriptor", syntax proto3)
+// tslint:disable
+import { MessageType } from "@protobuf-ts/runtime";
+/**
+ * @generated from protobuf message peerDescriptor.PeerDescriptor
+ */
+export interface PeerDescriptor {
```

Comment:
The problem is that we need to do define the PeerDescriptor to the protobuf messages in the .proto files. This has the side effect of generating the files.

Reply:
Maybe easiest to generate, but then delete file:
```sh
sed -i '' 's|import { PeerDescriptor } from "../../dht/protos/PeerDescriptor";|import { PeerDescriptor } from "@streamr/dht";|' generated/packages/trackerless-network/protos/NetworkRpc.ts
rm generated/packages/dht/protos/PeerDescriptor.ts
```

Adding these two lines to `proto.sh` solves the issue.

Reply:
Can't build the generated NetworkRpc files if PeerDescriptor is removed from the proto build

Reply:
What error message you get?

For me e.g. this runs all ok:
```
cd packages/trackerless-network
./proto.sh
npm run build
```

(did `npm run clean && npm run bootstrap` before this, so should the workspace should be clean)

Could that error be caused by other changes? When I DIDN't include this two line snippet, and deleted the generated directory in `trackerless-network,` I got some errors like this when doing `npm run build`:
Cannot find module '../generated/packages/trackerless-network/protos/NetworkRpc' or its corresponding type declarations.
8 import { ProxyDirection, StreamMessage } from '../generated/packages/trackerless-network/protos/NetworkRpc'

Reply:
I mean I suppose we could just stab the imports but it seems too hacky to do so. Especially since we persist the generated files only to get around a known issue in the ts-protobuf library (https://github.com/timostamm/protobuf-ts/issues/270).

Reply:
I'll try to fix this completely in https://linear.app/streamr/issue/NET-1371/fix-problem-with-building-protobuf-files-in-linux-to-get-rid-of